frontend/app/administrator/settings/details/page.tsx

```
@@ -209,3 +219,52 @@ export default function Details() {
+        <div className="space-y-2">
+          <Skeleton className="h-4 w-12" />
+          <Skeleton className="h-10 w-full" />
+        </div>
+        <div className="space-y-2">
+          <Skeleton className="h-4 w-20" />
+          <Skeleton className="h-10 w-full" />
+        </div>
+      </div>
+
+      <Skeleton className="h-4 w-[280px]" />
+      <Skeleton className="h-10 w-32 rounded-md mt-6" />
```

ESLint

[error] Replace `h-10·w-32·rounded-md·mt-6` with `mt-6·h-10·w-32·rounded-md`

(prettier/prettier)